packages/eslint-plugin/src/rules/jsx-pragma/__tests__/rule.test.ts

```diff
@@ -300,30 +332,7 @@ import { useState } from 'react';
       ],
     },
     {
-      code: `
```

Comment:
deleted this test case as it appears to be a duplicate of the previous one (albeit with different spacing that doesn't actually impact eslint's AST parsing)